README.md

```diff
@@ -21,6 +21,7 @@ Compatibility matrix:
 
 Metrics Server | Metrics API group/version | Supported Kubernetes version
 ---------------|---------------------------|-----------------------------
+0.3.x          | `metrics.k8s.io/v1beta1`  | 1.8+
```

Comment:
I think new release dropped support for unsecured kubelet port, this also adds requirement for kubelet to support Token Review. 
Will this change supported versions?
For example GKE does not support Token Review yet (I'm working on it, but still don't have confirmation from mikedanese)

Reply:
If we consider rolling this version into k8s 1.12, I would prefer to have https://github.com/kubernetes-incubator/metrics-server/commit/9bb8fe9d4c09e5f5ea78e244b188ca60aef8fffa rolled back. 

We can change default behavior to use secure port, but we should not drop support for http.

Reply:
> Will this change supported versions?

It shouldn't change the supported versions, since Kubernetes has supported kubelet token review for a while.

Plus, it doesn't require token review -- presumably, you can use kubeconfig with client certs instead.  Is that insufficient?

Reply:
Can you provide configuration for GKE in deployment?

Reply:
talked with @serathius on slack a bit, and the upshot is that if GKE doesn't have Kubelet token support, configuration for the automated GKE tests in k/k is going to be tricky.  I'm going to add the option back in, with the expectation that we'll get a timeline for it's removal in place soon.

Reply:
https://github.com/kubernetes-incubator/metrics-server/pull/116